smoketest/tests/v2/send_ena_to_penpal.rs

```diff
@@ -109,10 +109,9 @@ async fn send_token_to_penpal_v2() {
 			SetTopic(H256::random().into()),
 		]),
 	}]));
-	let encoded_xcm = message.encode();
 
-	let xcm = Bytes::from(encoded_xcm);
-	let claimer = Bytes::from(vec![]);
+	let xcm = Bytes::from(message.encode());
+	let claimer = Bytes::from(beneficiary().encode());
```

Comment:
The claimer should be set with a 32-byte address, or the test will fail since the default H160 claimer does not work on AH.

Reply:
Cool, I think is fine for now. We discussed in the call on Friday to set the Snowbridge sovereign as the claimer, if the claimer is not provided.